sdk/src/signer/mod.rs

```diff
@@ -1,244 +1,9 @@
-//! Abstractions and implementations for transaction signers.
-
 #![cfg(feature = "full")]
-
-use {
-    crate::{
-        pubkey::Pubkey,
-        signature::{PresignerError, Signature},
+pub use {
```

Comment:
Do we need the deprecation message here?

Reply:
Done with [06426cc](https://github.com/anza-xyz/agave/pull/3087/commits/06426cc53236039eca9aac6569d6de62c1a569d2)